src/utils/array_utils.F90.in

```diff
@@ -39,15 +40,13 @@ contains
     ! Returns the indices of the maximum k values in data, in sorted order (so the first
     ! index will be the largest, the second index will be the second largest, etc.)
     !
-    ! The output array (max_indices) is allocated here
-    !
-    ! This is O(n*k) (where n is the size of the input array)
```

Comment:
Please restore the comment about this routine being O(n*k) - this gives an estimate of the speed of this routine.

Reply:
Ok, changing to
`! Speed estimate: O(n*k) (where n is the size of the input array)`
to make clearer. I hope that's ok.